src/main/java/lotto_auto/model/LottoNumber.java

```diff
@@ -0,0 +1,64 @@
+package lotto_auto.model;
+
+import lotto_auto.ErrorMessage;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoNumber {
+
+    private static final Integer LOTTO_NUMBER_COUNT = 6;
+    private static final Integer LOTTO_NUMBER_MIN_RANGE = 0;
+    private static final Integer LOTTO_NUMBER_MAX_RANGE = 45;
+    private List<Integer> lottoNumber;
+
+    public LottoNumber(List<Integer> lottoNumber) {
+        ThrowIfNull(lottoNumber);
+        ThrowIfInValidLottoNumberCount(lottoNumber);
+        ThrowIfInvalidLottoNumberRange(lottoNumber);
+        ThrowIfDuplicate(lottoNumber);
+        this.lottoNumber = lottoNumber;
+    }
+
+    public List<Integer> export() {
+        return this.lottoNumber;
```

Comment:
Collections.unmodifiablelist()를 활용하여 외부에서 컬렉션을 건드리지 못하도록 read-only로 리턴하면 좋을 것 같아요!
```suggestion
        return Collections.unmodifiablelist(this.lottoNumber);
```

Reply:
!! 좋은 방법이네요 일급 컬렉션을 생성하고 어쩔수 없이 export 를 생성했는데
unmodifiablelist 를 사용하면 외부에서 건드리지 못하도록 할수있겠네요